File: web/lib/domain/danWei.dart
//单位类
class DanWei {
  final String id;
  final String xiaqu; //辖区
  final String name; //单位名称

  //({this.id, this.type, this.name, this.zccl, this.value});
  DanWei(this.id, this.xiaqu, this.name);

  DanWei.fromJson(Map<String, dynamic> map)
      : id = map['id'],
        xiaqu = map['xiaqu'],
        name = map['name'];

  Map<String, dynamic> toJson() => {'id': id, 'type': xiaqu, 'name': name};
}

File: mongodb/src/main/java/com/yw/xypj2/domain/DanWei.java
/*
 * Copyright (c) 2018，版权归作者所有。
 * 项目名称：xypj
 * 文件名称：  Pinjiazhibiao.java
 * 日期： 18-3-1 下午9:35
 * 作者：闫伟 15357668379 563180182@qq.com
 */

package com.yw.xypj2.domain;

import org.springframework.data.mongodb.core.mapping.Document;

//单位类
@Document
public class DanWei extends Shiti {

    private String xiaqu;
    private String name;

    public DanWei() {
        super();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getXiaqu() {
        return xiaqu;
    }

    public void setXiaqu(String xiaqu) {
        this.xiaqu = xiaqu;
    }

}
